### PR TITLE
Task duration

### PR DESCRIPTION
**Pull request by whyte624**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Yes
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | #526

### CODE REVIEW

File: src/Executor/Informer.php [hunk 1]
```diff
@@ -51,7 +57,12 @@ public function endTask()
         if ($this->output->getVerbosity() == OutputInterface::VERBOSITY_NORMAL && !$this->output->getWasWritten()) {
             $this->output->write("\033[k\033[1A\r<info>✔</info>\n");
         } else {
-            $this->output->writeln("<info>✔</info> Ok");
+            $endTime = round(microtime(true) * 1000);
+            $millis = $endTime - $this->startTime;
+            $seconds = floor($millis / 1000);
+            $millis = $millis - $seconds * 1000;
+            $taskTime = ($seconds > 0 ? "{$seconds}s " : "")."{$millis}ms";
+            $this->output->writeln("<info>✔</info> Ok after {$taskTime}");
```
Comment (elfet) [inline src/Executor/Informer.php]: May be add some format? Like gray colors: `<info>✔</info> Ok <gray>[$taskTime]</gray>`

Comment (whyte624) [inline src/Executor/Informer.php]: Is there such tag as <gray> in output? When I tested it looked fine. Anyway, if you insist, I can write something like `<fg=gray>[$taskTime]</>`.

Comment (elfet) [inline src/Executor/Informer.php]: Ok!

**Comment by elfet** [comment]
And please rebase.

File: src/Executor/Informer.php (hunk 1 above)
Comment (whyte624) [inline src/Executor/Informer.php]: Unfortunately there is no gray color for output. So, I've added just square brackets.

**Pull request merged**